project_timesheet_time_control/security/project_security.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data noupdate="0">
+
+    <record id="account_analytic_line_visibility_rule" model="ir.rule">
+        <field name="name">Account Analytic Line: employees: public or employee or following or assigned</field>
+        <field name="model_id" ref="model_account_analytic_line"/>
+        <field name="domain_force">['|',
+                                        ('user_id', '=', user.id),
+                                        '|',
+                                        ('task_id.user_id', '=', user.id),
+                                        '|',
```

Comment:
Convention seems to be starting the domain by all the ORs `['|', '|', '|', ...]`. It works the same anyway, so nevermind, but please fix the indentation here, is quite confusing.